### PR TITLE
Issues 13, 14

### PR DESCRIPTION
**Pull request by cnstlungu**
#13 , #14 

Switch map to run from APIs
Add sidebar with building information
Add map locations filtering based on seismic risk

Additional changes:
- Fix an_construire

Please review.

### CODE REVIEW

**Comment by cnstlungu** [comment]
@RaduCStefanescu  @costibleotu 

I would expect also the following changes to also be required:
- js script to be built for production/minified
- set up npm (currently using the mapping library as a linked script)
- review CORS setup
- organize JS using best practices in a Django project

Let me know if you want to treat it as part of this issue or for a later one.

Would be great if I could chat with someone more experienced with Django + FrontEnd to iron out the rough edges.

**Comment by costibleotu** [comment]
I wolud say we do another task for the JS.

We can have a call and I can propse some settings. 
I normally user Django + Webpack (for bundle and minification)

File: seismic_site/seismic_site/settings.py [hunk 1]
```diff
@@ -91,8 +91,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'risc_db',
-        'USER': 'seism',
-        'PASSWORD': 'seism',
+        'USER': 'postgres',
+        'PASSWORD': 'postgres',
```
Comment (tudoramariei) [inline seismic_site/seismic_site/settings.py]: Made a change in the README (see PR #35) and the command for setting up the Postgres DB uses:
`POSTGRES_USER=seism -e POSTGRES_PASSWORD=seism`
So these changes aren't necessary anymore

**Comment by tudoramariei** [comment]
After testing the changes from this PR, it seems that it fixes #10 #13 #14 #31 #32 
It directly fixes #13 and #14 through the changes and new features
It resolves some issues with the code and thus fixes #10 #31 and #32 and makes them properly work

**Comment by tudoramariei** [comment]
Since some issues were still unsolved, PR #44 was created and merged together with these changes
Since I made a squash merge, I mentioned you, @cnstlungu in the description
Thanks a lot for this contribution!

**Pull request closed without merging**